tasks/test_syntax.js

```diff
@@ -177,7 +177,7 @@ function assertCircularDeps() {
         var logs = [];
 
         // see https://github.com/plotly/plotly.js/milestone/9
-        var MAX_ALLOWED_CIRCULAR_DEPS = 12;
+        var MAX_ALLOWED_CIRCULAR_DEPS = 17;
```

Comment:
I got a little too excited in https://github.com/plotly/plotly.js/pull/1613, that PR didn't remove any circular deps. It only made `madge` think that there were less ciruclar deps (their algo must order-dependent).

If you don't believe me, try adding `var Fx = require('../fx');` in `annotations_defaults` like [here](https://github.com/plotly/plotly.js/pull/1582/commits/f986fd2e34afa4fe625a995831888899d9b2442b#diff-81cae611e22b976932e957a99623ac1eL14) and you get back the 12 circular deps of PR #1613 .